<ui version="4.0" >
 <class>GexDbPlugin_StTdb_CfgWizard_Base</class>
 <widget class="Q3Wizard" name="GexDbPlugin_StTdb_CfgWizard_Base" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Database Link Wizard</string>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="pageDbConnection" >
   <attribute name="title" >
    <string>Database connection settings</string>
   </attribute>
   <layout class="QVBoxLayout" >
    <property name="spacing" >
     <number>15</number>
    </property>
    <item>
     <layout class="QGridLayout" >
      <item row="0" column="1" >
       <widget class="QLineEdit" name="editDbHostNameIP" >
        <property name="toolTip" >
         <string>Enter the computer name where the corporate database is hosted</string>
        </property>
        <property name="maxLength" >
         <number>80</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="labelDbType" >
        <property name="text" >
         <string>DB Type:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>comboBoxDbType</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="labelDbPassword" >
        <property name="text" >
         <string>Password:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>editDbPassword</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QSpinBox" name="spinBoxDbPortNb" >
        <property name="maximum" >
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="labelDbHostNameIP" >
        <property name="text" >
         <string>Host name / IP:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>editDbHostNameIP</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="labelDbName" >
        <property name="text" >
         <string>DB name:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>editDbName</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="labelDbPortNb" >
        <property name="text" >
         <string>Port:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>spinBoxDbPortNb</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="labeDbUserName" >
        <property name="text" >
         <string>User name:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>editDbUserName</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="comboBoxDbType" >
        <property name="toolTip" >
         <string>Define the driver type required to connect to your corporate database</string>
        </property>
        <item>
         <property name="text" >
          <string>OCI</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>MySQL</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QLineEdit" name="editDbPassword" >
        <property name="toolTip" >
         <string>Enter the database password required at connection</string>
        </property>
        <property name="maxLength" >
         <number>80</number>
        </property>
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="editDbUserName" >
        <property name="toolTip" >
         <string>Enter the database user name required at connection</string>
        </property>
        <property name="maxLength" >
         <number>80</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QLineEdit" name="editDbName" >
        <property name="toolTip" >
         <string>Enter the database name to link to</string>
        </property>
        <property name="maxLength" >
         <number>80</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="spacer9" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>110</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="Line" name="line4" >
      <property name="frameShape" >
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pageDbConnection_labelDbStatus" >
      <property name="text" >
       <string>Database status (errors...)</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="pageTableSelection" >
   <property name="geometry" >
    <rect>
     <x>6</x>
     <y>36</y>
     <width>521</width>
     <height>310</height>
    </rect>
   </property>
   <attribute name="title" >
    <string>Table selection</string>
   </attribute>
   <layout class="QVBoxLayout" >
    <property name="spacing" >
     <number>15</number>
    </property>
    <item>
     <widget class="Q3ListBox" name="listBoxTables" />
    </item>
    <item>
     <widget class="Line" name="line3" >
      <property name="frameShape" >
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pageTableSelection_labelDbStatus" >
      <property name="text" >
       <string>Database status (errors...)</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="pageDbMapping" >
   <property name="geometry" >
    <rect>
     <x>6</x>
     <y>36</y>
     <width>521</width>
     <height>310</height>
    </rect>
   </property>
   <attribute name="title" >
    <string>Field mapping</string>
   </attribute>
   <layout class="QVBoxLayout" >
    <property name="spacing" >
     <number>15</number>
    </property>
    <item>
     <widget class="QLabel" name="labelMapping" >
      <property name="text" >
       <string>Fields mapping for: TBL_FILES</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Q3ListView" name="listViewMapping" >
      <property name="toolTip" >
       <string>Double-click any Key entry to edit...</string>
      </property>
      <property name="showSortIndicator" >
       <bool>true</bool>
      </property>
      <column>
       <property name="text" >
        <string>Corporate DB Key</string>
       </property>
       <property name="clickable" >
        <bool>true</bool>
       </property>
       <property name="resizable" >
        <bool>true</bool>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Galaxy Key</string>
       </property>
       <property name="clickable" >
        <bool>true</bool>
       </property>
       <property name="resizable" >
        <bool>true</bool>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line5" >
      <property name="frameShape" >
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pageDbMapping_labelDbStatus" >
      <property name="text" >
       <string>Database status (errors...)</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="pageFtpSettings" >
   <property name="geometry" >
    <rect>
     <x>6</x>
     <y>36</y>
     <width>521</width>
     <height>310</height>
    </rect>
   </property>
   <attribute name="title" >
    <string>Ftp settings</string>
   </attribute>
   <layout class="QVBoxLayout" >
    <property name="spacing" >
     <number>15</number>
    </property>
    <item>
     <layout class="QGridLayout" >
      <item row="2" column="1" >
       <widget class="QLineEdit" name="editFtpUserName" />
      </item>
      <item row="0" column="3" >
       <widget class="QSpinBox" name="spinBoxFtpPortNb" >
        <property name="maximum" >
         <number>9999</number>
        </property>
        <property name="value" >
         <number>21</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="editFtpHostNameIP" />
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="labelFtpPassword" >
        <property name="text" >
         <string>Password:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>editFtpPassword</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="labelFtpUserName" >
        <property name="text" >
         <string>User name:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>editFtpUserName</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QCheckBox" name="checkBoxFtpHostNameFromDB" >
        <property name="text" >
         <string>From Database</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="labelFtpPortNb" >
        <property name="text" >
         <string>Port:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>spinBoxFtpPortNb</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="labelFtpHostNameIP" >
        <property name="text" >
         <string>Host name / IP:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>editFtpHostNameIP</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QLineEdit" name="editFtpPassword" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="spacer10" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>151</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>Q3Wizard</class>
   <extends>QDialog</extends>
   <header>q3wizard.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>Q3Frame</extends>
   <header>q3listbox.h</header>
  </customwidget>
  <customwidget>
   <class>Q3ListView</class>
   <extends>Q3Frame</extends>
   <header>q3listview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>editDbHostNameIP</tabstop>
  <tabstop>spinBoxDbPortNb</tabstop>
  <tabstop>comboBoxDbType</tabstop>
  <tabstop>editDbName</tabstop>
  <tabstop>editDbUserName</tabstop>
  <tabstop>editDbPassword</tabstop>
  <tabstop>listViewMapping</tabstop>
  <tabstop>editFtpHostNameIP</tabstop>
  <tabstop>spinBoxFtpPortNb</tabstop>
  <tabstop>checkBoxFtpHostNameFromDB</tabstop>
  <tabstop>editFtpUserName</tabstop>
  <tabstop>editFtpPassword</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
